tree/20_cfe_basics/ncf_lib.cf

```diff
@@ -861,31 +839,34 @@ body classes classes_generic_return_boolean_list_two(x, y, boolean_prefix, true_
     kept_returncodes => { @{true_return_codes} };
     repaired_returncodes => { @{false_return_codes} };
 
+    inherit_from => classes_generic_two("${x}", "${y}");
+
     promise_repaired => { "promise_kept_$(x)", "$(x)_kept", "$(x)_ok", "$(x)_not_repaired", "$(x)_reached",
```

Comment:
why keeping this one?

Reply:
We need the "${boolean_prefix}_false" one, and I am pretty sure that cfengine does not allow merging of body attributes but override only.